PyDSS/dssInstance.py

```diff
@@ -502,10 +502,10 @@ def RunSimulation(self, project, scenario, MC_scenario_number=None):
                 if within_range:
                     pydss_has_converged = self.RunStep(step)
                     opendss_has_converged = dss.Solution.Converged()
-                    if not opendss_has_converged:
-                        self._Logger.error("OpenDSS did not converge at step=%s pydss_converged=%s",
-                                            step, pydss_has_converged)
-                        self._HandleOpenDSSConvergenceErrorChecks(step)
+                    # if not opendss_has_converged:
```

Comment:
Is this and other changes just for debugging?

Reply:
Yes they are I will rever these changes